.github/workflows/zip-templates-and-add-to-release.yml

```
@@ -0,0 +1,29 @@
+name: zip-templates-and-add-to-release
+on:
+  release:
+    types: [published]
+jobs:
+  add-templates-to-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install dependencies
+        run: |
+          npm install -g yarn
+          sudo apt-get update && sudo apt-get install -y --allow-unauthenticated zip
+          yarn install
+      - name: Build templates
+        run: |
+          cd src
+          RELEASE_VERSION=$GITHUB_REF yarn cdn-bundle
```

`GITHUB_REF` will be `ref/tags/<version>`, this needs to be changed to:
```suggestion
          RELEASE_VERSION=${{ github.event.release.tag_name }} yarn cdn-bundle
```